dstep/Configuration.d

```diff
@@ -34,8 +37,11 @@ struct Configuration
     /// array of parameters needed to be forwarded to clang driver
     string[] clangParams;
 
-    /// output file name or folder (in case there are many input files)
-    string output;
+    /// output file name or folder
+    string outputPath;
+
+    /// in case there are many input files or dir output option provided
+    bool isOutputToDir;
```

Comment:
Is the intention that both `-O` and `-o` are specified since this is a bool?

Reply:
Yes